snapshot/snapshot.go

```diff
@@ -378,22 +370,12 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 	return mounts, err
 }
 
-func (o *snapshotter) findReferrerLayer(ctx context.Context, key string) (string, snapshots.Info, error) {
-	return snapshot.IterateParentSnapshots(ctx, o.ms, key, func(id string, info snapshots.Info) bool {
-		return o.fs.CheckReferrer(ctx, info.Labels)
-	})
-}
-
-func (o *snapshotter) findMetaLayer(ctx context.Context, key string) (string, snapshots.Info, error) {
-	return snapshot.IterateParentSnapshots(ctx, o.ms, key, func(id string, i snapshots.Info) bool {
-		return label.IsNydusMetaLayer(i.Labels)
-	})
-}
-
 // The work on supporting View operation for nydus-snapshotter is divided into 2 parts:
 // 1. View on the topmost layer of nydus images or zran images
 // 2. View on the any layer of nydus images or zran images
 func (o *snapshotter) View(ctx context.Context, key, parent string, opts ...snapshots.Opt) ([]mount.Mount, error) {
+	log.L.Debugf("[View] snapshot with key %s, parent %s", key, parent)
```

Comment:
Why duplicate one log line with L417?

Reply:
One is debug, another one is info. And print debug message at the entry for trouble shooting.

Reply:
removed